C++/Env/Dev.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _DEV_H_
+#define _DEV_H_
+
+#include "Env.h"
+
+class Dev : public Env
+{
+public:
+  std::string getEnv (void);
+
+private:
+  constexpr static const char * env = "DEVELOPMENT";
+};
+
+#endif
```

Comment:
I find #endif // MACRO to be a good convention as it shows the end of the associated ifndef statement